### PR TITLE
MRG: Added Yeo dataset description

### PR DESCRIPTION
**Pull request by bthirion**
Added an initial description of the YEO dataset. Addresses #337 

WIP:
* integration into doc
* renaming of fields/variables

### CODE REVIEW

**Comment by bthirion** [comment]
Should be ready for review

File: nilearn/datasets.py [hunk 1]
```diff
@@ -792,10 +792,10 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     data: sklearn.datasets.base.Bunch
         dictionary-like object, keys are:
 
-        - "tight_7", "liberal_7": 7-region parcellations, resp. tightly
-          fitted to cortex shape, and liberally fitted.
+        - "thin_7", "thick_7": 7-region parcellations,
```
Comment (AlexandreAbraham) [inline nilearn/datasets.py]: I agree that these names are better but shouldn't we stick to the semantic used in the paper?

File: examples/plot_atlas.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Plot the regions of a reference atlas.
+"""
+
+import matplotlib.pyplot as plt
+from nilearn.datasets import fetch_yeo_2011_atlas
+from nilearn import plotting
+
+atlas = fetch_yeo_2011_atlas()
+print atlas.DESCRIPTION
```
Comment (banilo) [inline examples/plot_atlas.py]: Isn't it rare to use capital-letter-only variable names in nilearn?

File: nilearn/datasets.py (hunk 1 above)
Comment (bthirion) [inline nilearn/datasets.py]: I had a  quick look at the paper, and did not find these names. However, they are present in the following page
http://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation_Yeo2011
I do not have any strong opinion: the names are probably slightly better. Do we need to have some kind of compatibility with the freesurfer site ?

File: examples/plot_atlas.py (hunk 2 above)
Comment (GaelVaroquaux) [inline examples/plot_atlas.py]: After discussion with the group, we feel that it was better renaming this variable to lowercase.

I am implementing this change right now and merging. Thanks @bthirion

**Comment by GaelVaroquaux** [comment]
Merged. Thank you!

**Pull request closed without merging**